server/index.js

```diff
@@ -304,6 +305,63 @@ app.delete('/api/favorite/allSavedFavorites/:favoriteId', (req, res, next) => {
 
 });
 
+// search users
+
+app.get('/api/users/search', (req, res, next) => {
+  const { userId } = req.user;
+  if (!userId) {
+    return;
+  }
+  const { 'location-info': location, 'search-type': searchType } = req.headers;
```

Comment:
You can get the query parameters with `req.query`.

https://expressjs.com/en/4x/api.html#req.query